src/lib/core/a11y/focus-trap.ts

```diff
@@ -1,6 +1,8 @@
 import {Component, ViewEncapsulation, ViewChild, ElementRef} from '@angular/core';
 import {InteractivityChecker} from './interactivity-checker';
 
+/** Selector for nodes that should have a higher priority when looking for focus targets. */
+const FOCUS_TARGET_SELECTOR = '[md-focus-target]';
```

Comment:
I'd still like to have separate `md-focus-start` and `md-focus-end` attributes, even if they're not directives. With only one attribute, users either have to specify it exactly zero times or more than one time- a user just trying to set where the focus should start would also inadvertently be setting where the focus should wrap backwards to.